Gemfile

```diff
@@ -45,7 +45,7 @@ group :production do
 end
 
 # administration
-gem 'administrate'
+gem 'administrate', github: 'thoughtbot/administrate'
```

Comment:
Administrate doesn't have a release that will `bundle install` when using rails 6, but master does work.  They have a plan to release a version sometime soon to fix this.